deployment/tfhub/object-detection/base64/object-detection-base64-deploy.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Object Identification (TensorFlow Hub) - Base64 Images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook identifies whether a given object is present in a set of images. If the object is found, it returns a dataframe with the file name, score and bounding boxes.\n",
+    "We are using a dataset from [UCF](https://www.crcv.ucf.edu/data/GMCP_Geolocalization/#Dataset) that contains 62,058 high quality Google Street View images, and [TensorFlow Hub](https://www.tensorflow.org/hub), a library and platform for transfer learning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook has been tested with Python 3.10.6 and the following package versions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install dill==0.3.6\n",
+    "!pip install Pillow==9.3.0\n",
```

Comment:
We can do this as a follow-up later, after merging:

Colab seems to use `Pillow==7.1.2` by default, and installing `Pillow==9.3.0` requires restarting the runtime, which is a bit tedious. I wonder if it would be possible to change our model a bit to be compatible with `Pillow==7.1.2` so we do not have to install the newer version 🤔

Reply:
Exactly. Running with `Pillow==7.1.2` Colab throws an error about Pillow's `Image.Resampling`. That's why I needed the latest stable version.
I'll try to find a workaround so we can run without restarting the runtime :+1:

Reply:
Yeah, I wonder if `7.1.2` has something similar to `9.3.0`'s `Image.Resampling` that we could use instead 🤔 But maybe not

Reply:
Found a solution, but it works only with Python 8. Commited in https://github.com/VertaAI/examples/tree/tfhub-object-detection-improvements